### PR TITLE
Update sig-docs-hi members

### PR DESCRIPTION
**Pull request by anubha-v-ardhan**
Owners and Reviewers for Hindi localization team.

cc: 
@sftim 
@divya-mohan0209 
@mittalyashu

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Thanks
/lgtm

**Comment by sftim** [comment]
This pairs with https://github.com/kubernetes/website/pull/29666

**Comment by sftim** [comment]
@anubha-v-ardhan if you're able to squash this down to 1 commit, that'd be handy.

**Comment by anubha-v-ardhan** [comment]
> @anubha-v-ardhan if you're able to squash this down to 1 commit, that'd be handy.

Done :+1:

File: config/kubernetes/sig-docs/teams.yaml [hunk 1]
```diff
@@ -83,18 +83,18 @@ teams:
     - yastij
     privacy: closed
   sig-docs-hi-owners:
-    description: Owners for Hindi content
+    description: Approvers for Hindi content
     members:
-    - daminisatya
+    - anubha-v-ardhan
+    - divya-mohan0209
```
Comment (mittalyashu) [inline config/kubernetes/sig-docs/teams.yaml:89]:
```suggestion
    - divya-mohan0209
```

Is she an active contributor? I didn't saw any PR in kubernetes/website repo.

Comment (anubha-v-ardhan) [reply config/kubernetes/sig-docs/teams.yaml:89]: Yes, Divya is a very active contributor doing a lot of good work, also recently [nominated for SIG Docs Co-Chair](https://groups.google.com/g/kubernetes-sig-docs/c/AFWmzt43G3Y?pli=1).
We also had an [onofficial poll on #kubernetes-docs-hi Slack channel](https://kubernetes.slack.com/archives/CJ14B9BDJ/p1631109069025400?thread_ts=1629833966.493300&cid=CJ14B9BDJ) to which almost almost all of our active contributors including Tim gave +1.

Comment (anubha-v-ardhan) [reply config/kubernetes/sig-docs/teams.yaml:89]: Also, Kunal will join as reviewer later once he becomes org member officially.

Comment (anubha-v-ardhan) [reply config/kubernetes/sig-docs/teams.yaml:89]: Yes, Divya is a very active contributor doing a lot of good work, she has been involved with SIG-Docs for few years  and was also a Docs shadow for 1.19. She recently got [nominated for SIG Docs Co-Chair](https://groups.google.com/g/kubernetes-sig-docs/c/AFWmzt43G3Y?pli=1) too.

We also had an [unofficial poll on #kubernetes-docs-hi Slack channel](https://kubernetes.slack.com/archives/CJ14B9BDJ/p1631109069025400?thread_ts=1629833966.493300&cid=CJ14B9BDJ) to which almost all of our active contributors including Tim gave +1.

**Pull request merged**